app/controllers/rounds_controller.rb

```diff
@@ -40,6 +40,46 @@ def update
     end
   end
 
+  def activate
+    @game = Game.find_by(id: params[:id])
+    @round = Round.find_by(id: params[:id])
+    notice = "Round #{@round.category} "
+    if  @round.active != true
+      @round.update(active: true)
+      notice.concat("actived")
+    else
+      notice.concat("is already actived")
+    end
+    redirect_to @game, notice: notice
+  end
+
+  def deactivate
+    @game = Game.find_by(id: params[:id])
```

Comment:
Same problem as `activate`
```suggestion
    @game = Game.find_by(id: params[:game_id])
```